### ISSUE TITLE
invalid score null for color-contrast

### ISSUE THREAD

**Issue by connorjclark** [GoogleChrome/lighthouse#10644]
Repro (master):

```
node lighthouse-cli https://www.coursehero.com/file/11/030507/ --view --only-audits=color-contrast
```

![image](https://user-images.githubusercontent.com/4071474/80416906-bb26ca00-8889-11ea-900e-3acc951fe5ec.png)

You can see the true error with a breakpoint in the axe gatherer:

![image](https://user-images.githubusercontent.com/4071474/80417061-017c2900-888a-11ea-8b6f-b54ba71d8f1e.png)

Having trouble reproducing without Lighthouse. Also the error doesn't happen for `--emulated-form-factor none`

Should we use `.stack` for making the error message here?

axe bug: https://github.com/dequelabs/axe-core/issues/2178

**Comment by patrickhulce** [comment]
Dibs on the scoreDisplayMode issue fix :)

**Comment by paulirish** [comment]
if axe 5.3.0 fixes this then #10637 will address this?
